.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2
+jobs:
+  bucklescript-build-and-test:
+    docker:
+      - image: circleci/node:11
+
+    environment:
+      CI: true
+      NODE_ENV: test
+
+    working_directory: ~/repo/bs
+    steps:
+      - checkout:
+          path: ~/repo
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - v1-dependencies-{{ .Branch }}-
+            - v1-dependencies-
+      - run: npm install
+      - run: npm run build
+      - run: npm test
+      - save_cache:
+          key: v1-dependencies-{{ .Branch }}-{{ checksum "package-lock.json" }}
+          paths:
+            - ~/repo/bs/node_modules
+
+  native-build-and-test:
+    docker:
+      - image: ocaml/opam2
+    environment:
+      CI: true
+    working_directory: ~/repo/native
+    steps:
+      - checkout:
+          path: ~/repo
+      - restore_cache:
+          keys:
+            - v1-native-dependencies-{{ .Branch }}-{{ checksum "tablecloth-native.opam" }}
+            - v1-native-dependencies-{{ .Branch }}-
+            - v1-native-dependencies-
+      # m4 is a system dependency required by conf-m4 -> ocamlfind -> fmt -> alcotest
+      - run: sudo apt-get install -y m4
+      - run: opam update && opam install alcotest base dune junit junit_alcotest -y
+      - run: opam config exec -- dune build
+      - run: opam config exec -- dune runtest -f
+      - save_cache:
+          key: v1-native-dependencies-{{ .Branch }}-{{ checksum "native/tablecloth-native.opam" }}
```

Comment:
Looks like saving the cache doesnt work: https://circleci.com/gh/Dean177/tablecloth/13